CHANGELOG.md

```diff
@@ -4,6 +4,90 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.0] - 2024-07-23
+
+### Migrating from 0.4.0 -> 0.6.0
+
+Module Imports
+
+```
+use deepgram::{
+---    transcription::prerecorded::{
++++    common::{
+        audio_source::AudioSource,
+        options::{Language, Options},
+    },
+    Deepgram, DeepgramError,
+};
+```
+
+Streaming Changes
+
+Now you can pass Options using stream_request_with_options
+```
+let options = Options::builder()
+    .smart_format(true)
+    .language(Language::en_US)
+    .build();
+
+let mut results = dg
+    .transcription()
+    .stream_request_with_options(Some(&options))
+    .file(PATH_TO_FILE, AUDIO_CHUNK_SIZE, Duration::from_millis(16))
+    .await?
+    .start()
+    .await?;
+```
+
+### Changed
+
+- Add streaming features
+- Add support for pre-recorded features when streaming
+- Add Speech to Text
+- Refactor Code
```

Comment:
Say "reorganize" instead of refactor.  A refactor should technically be an invisible change that only affects the internal structure of a library, not a user facing change.